fireant/queries/builder/dataset_blender_query_builder.py

```diff
@@ -79,7 +78,6 @@ def _datasets_and_field_maps(blender):
     def _flatten_blend_datasets(dataset) -> List:
         primary = dataset.primary_dataset
         secondary = dataset.secondary_dataset
```

Comment:
Saw the comment got removed. Does this need explaining in code comments?

Reply:
The method it calls has a huge comment description. So I think Tim just forgot to remove the TODO.